docs/next/guides/getting-started/installation.md

```diff
@@ -11,20 +11,65 @@ tags:
 
 [[toc]]
 
-## Overview
-This guide details how to install Handsontable.
+Install Handsontable in your preferred way.
 
 ::: tip
-This section is dedicated to the pure JavaScript version of Handsontable. If you use a framework in your project, follow one of the available guides to install and use the library:
- - [Integrate with React](@/guides/integrate-with-react/react-installation.md)
- - [Integrate with Angular](@/guides/integrate-with-angular/angular-installation.md)
- - [Integrate with Vue 2](@/guides/integrate-with-vue/vue-installation.md)
- - [Integrate with Vue 3](@/guides/integrate-with-vue3/vue3-installation.md)
+To use Handsontable with a framework, see:
+
+ - [Installation in React](@/guides/integrate-with-react/react-installation.md)
+ - [Installation in Angular](@/guides/integrate-with-angular/angular-installation.md)
+ - [Installation in Vue 2](@/guides/integrate-with-vue/vue-installation.md)
+ - [Installation in Vue 3](@/guides/integrate-with-vue3/vue3-installation.md)
 :::
 
-## Download and install the library
+## Overview
+
+<code-group>
+  <code-block title="npm">
+
+  ```bash
+  npm install handsontable
+  ```
+
+  </code-block>
+  <code-block title="Your HTML">
+
+  ```html
+  <div id="your-container"></div>
```

Comment:
The container name "example" is a convention used on 37 pages. I would love to change it to "my-container", but only on all pages at once.

```suggestion
  <div id="example"></div>
```